### ISSUE TITLE
Lazy load layer

### ISSUE THREAD

**Issue by azizbekian** [moagrius/TileView#369]
Say I have this:
`tileView.addDetailLevel(1.000f, someFancyFormat, TILE_CELL_SIZE, TILE_CELL_SIZE);`

And my current scale is 0.500f
What I want is to load 1.000f images only when user zoomes that much, otherwise just do nothing. Conversely, stop loading that layer's images as soon as user zoomes out (say user zoomed in to 1.3f than zoomed out, we're no longer interested in that layer's tiles).
 
Is there such functionality implemented?

**Comment by moagrius** [comment]
you could have a detail level that just returned an empty (or transparent) bitmap.  You could also make the cell size as big as the entire TileView so it didn't make a bunch of them.

you'd need to create your own `BitmapDecoder` implementation, but you could probably just hack the existing one for assets - maybe just check if `data` is null, and if so return an empty or transparent bitmap.

you could also just hide it when it's smaller than a certain scale - you can hook into scale changes really easily. 
tl;dr is that there are probably a bunch of ways to do what you want, but there's no single method or flag you can turn on or off to do it.
